src/theme/bootstrap-override/bootstrap-italia/_chips.scss

```diff
@@ -2,7 +2,7 @@
 .public-ui .chip {
   display: inline-block;
   height: unset;
-  cursor: pointer;
+  cursor: default;
```

Comment:
Scusami, solita domanda di rito per scrupolo: hai provato semplicemente a togliere la riga invece di settare `default`, dato che è appunto il default? In altri termini, se la togli si vede con il default o ci sono altri stili che danno fastidio?

Reply:
Scusa, ti avevo già avvisato nel commento nella US
Sì, ho provato a toglierla e togliendola e basta, mettendo il cursore sul chip, il cursore diventa quello del "seleziona testo", con la stanghettina
![image](https://github.com/RedTurtle/design-comuni-plone-theme/assets/116291154/e39ab087-f129-41fc-a487-621357b58b5a)

Mettendo default, rimane la freccetta. Mi sembrava più corretto

Reply:
Ok va benissimo, grazie